README.md

```diff
@@ -1,6 +1,5 @@
 # Vet Clinic Database
 
-
 # 📗 Table of Contents
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to fix the broken links on your readme headers, like deployment, run tests, etc